src/main/java/forestry/plugins/PluginBuildCraftFuels.java

```diff
@@ -10,41 +10,57 @@
  ******************************************************************************/
 package forestry.plugins;
 
+import buildcraft.api.mj.MjAPI;
+import net.minecraftforge.fluids.Fluid;
+import net.minecraftforge.fluids.FluidRegistry;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fml.common.Optional;
+
+import buildcraft.api.fuels.BuildcraftFuelRegistry;
+import buildcraft.api.fuels.ICoolant;
+import buildcraft.api.fuels.ICoolantManager;
+import forestry.api.core.ForestryAPI;
 import forestry.api.modules.ForestryModule;
 import forestry.core.config.Constants;
+import forestry.core.fluids.Fluids;
 import forestry.core.utils.ModUtil;
 import forestry.modules.BlankForestryModule;
 import forestry.modules.ForestryModuleUids;
 
-// TODO: Buildcraft for 1.9
 @ForestryModule(containerID = ForestryCompatPlugins.ID, moduleID = ForestryModuleUids.BUILDCRAFT_FUELS, name = "BuildCraft 6 Fuels", author = "mezz", url = Constants.URL, unlocalizedDescription = "for.module.buildcraft6.description")
 public class PluginBuildCraftFuels extends BlankForestryModule {
 
 	@Override
 	public boolean isAvailable() {
-		return ModUtil.isAPILoaded("buildcraft.api.fuels", "[2.0, 3.0)");
+		return ModUtil.isModLoaded(Constants.BCLIB_MOD_ID);
```

Comment:
We should give this a version range so that it is inactive on the next major version.